_posts/2022-06-21-ElixirAHM22.md

```diff
@@ -0,0 +1,52 @@
+**ELIXIR All Hands 2022**
+
+During 6-9th of June, members of the European Galaxy team travelled to Amsterdam to join the [ELIXIR All Hands meeting 2022](https://elixir-europe.org/all_hands_2022).
+
+{% twitter https://twitter.com/BSerranoSolano/status/1533824511795142657?t=wIrbNbACZKjIUJC-HCUfzQ&s=09 %}
+
+There was a broad presence of Galaxy all over the 3 days of the meeting across the different workshops (parallel sessions) and plenary talks and discussions. We will highlight here the participants.
+
+{% twitter https://twitter.com/denbiOffice/status/1534872847126089728 %}
+
+On the first day; the 7th of June, Anup Kumar and Kaivan Kamali presented in the [AI-powered data analysis within the European Galaxy workshop](https://docs.google.com/document/d/17IVSvLmnianF6VOLO7GQfwPFDTpm9DF95ShHXnaarNo/edit#heading=h.8yacamuak08k) conducted using an [image classification tutorial](https://training.galaxyproject.org/training-material/topics/statistics/tutorials/CNN/tutorial.html). In the workshop, a convolutional neural network was trained to recognise handwritten digits using Galaxy Europe's deep learning tools in a stepwise manner. The steps included importing datasets, creating an architecture of the convolutional neural network, training it on imported train datasets and evaluating the trained model using test datasets and visualisation techniques. 
+
+{% twitter https://twitter.com/BSerranoSolano/status/1534113948135747584 %}
+
+On that same day afternoon, the ELIXIR-Galaxy community organised a workshop on [Earth Biodiversity](https://docs.google.com/presentation/d/1YvJsPK0DAcl2_pFwfCcYx9KnEM0eRyMcI82uOVUCZVg/edit) conducted by Frederik Coppens (ELIXIR-BE) and Björn Grüning (ELIXIR-DE). There were many talks, 7 in total,  including Engy Nasr’s presentation [slides: 83-102]. The workshop topic is highly relevant at the moment thanks to initiatives like the [Vertebrate Genomes Project (VGP)](https://vertebrategenomesproject.org/) and [European Reference Genome Atlas (ERGA)](https://erga-online.eu/), which will sequence and assemble the genomes of thousands of species over the next few years, and the plans that have been outlined in a submitted Horizon Europe call.
```

Comment:
```suggestion
On that same afternoon, the ELIXIR-Galaxy community organised a workshop on [Earth Biodiversity](https://docs.google.com/presentation/d/1YvJsPK0DAcl2_pFwfCcYx9KnEM0eRyMcI82uOVUCZVg/edit). The workshop topic is highly relevant at the moment thanks to initiatives like the [Vertebrate Genomes Project (VGP)](https://vertebrategenomesproject.org/) and [European Reference Genome Atlas (ERGA)](https://erga-online.eu/), which will sequence and assemble the genomes of thousands of species over the next few years, and the plans that have been outlined in a submitted Horizon Europe call.
```

Reply:
that was in the abstract I will remove it